BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m

```diff
@@ -513,7 +513,7 @@ - (void)setCurrentFolder:(BOXFolder *)folder
 - (void)switchToEmptyStateWithError:(NSError *)error
 {
     // No error means it's just an empty folder.
-    NSString *errorMessage = [[NSString alloc] init];
+    NSString *errorMessage;
```

Comment:
lets initialize to nil. It's good practice to always initialize local vars and for objects it's fine because they start nil, but for primitives you can still get junk values - better to be consistent so it's easy never to miss one.